pyanaconda/core/constants.py

```diff
@@ -390,3 +390,9 @@ class PayloadRequirementType(Enum):
 PARTITIONING_METHOD_MANUAL = "MANUAL"
 PARTITIONING_METHOD_INTERACTIVE = "INTERACTIVE"
 PARTITIONING_METHOD_BLIVET = "BLIVET"
+
+# Payload URL source types.
+URL_SOURCE_BASE = "BASE"
+URL_SOURCE_MIRRORLIST = "MIRRORLIST"
+URL_SOURCE_METALINK = "METALINK"
+URL_SOURCE_FILE = "FILE"
```

Comment:
This is confusing. I wouldn't mix protocols with types.

Reply:
Already removed :). Thanks for suggestion.